src/canvas.cpp

```diff
@@ -255,16 +262,68 @@ void Canvas::mouseReleaseEvent(QMouseEvent* event)
     }
 }
 
+QPointF Canvas::calcMr(QPoint p) {
```

Comment:
Would you mind changing this function name to be a bit more descriptive?

Reply:
Done.
Changed it to "changeMouseCoordinates"